sktime/data/datasets.py

```diff
@@ -45,3 +48,50 @@ def ellipsoids(laziness=0.97, seed=None):
         an object that contains methods to create discrete and continuous observations
     """
     return Ellipsoids(laziness=laziness, seed=seed)
+
+
+def pbf(n_burn_in=5000, n_jobs=None):
+    r""" Creates a position based fluids :cite:`macklin2013position` simulator.
+
+    The simulation box has dimensions :math:`[-40, 40]\times [-25, 25]` and the initial positions are positioned
+    around the top boundary of the box. The interaction distance is set to :math:`d = 1.5` and `n_burn_in` steps are
+    performed to equilibrate the system before returning the simulator.
+
+    For more details see :class:`PBF <sktime.data.PBF>`.
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import sktime
+
+        ftraj = sktime.data.pbf(n_burn_in=150).run(300)
+        f, axes = plt.subplots(3, 2, figsize=(15, 10))
+        for i, ax in enumerate(axes.flat):
+            ax.scatter(*(ftraj[i*50].reshape(-1, 2).T))
+            ax.set_title("t = {}".format(i*50))
+            ax.grid()
+        f.suptitle(r'PBF dataset observations.')
+        plt.show()
+
+    Parameters
+    ----------
+    n_burn_in : int, default=5000
+        Number of steps without any drift force to equilibrate the system.
+    n_jobs : int or None, default=None
+        Number of threads to use for simulation.
+
+    Returns
+    -------
+    pbf : PBF
+        The PBF simulator.
+    """
+    interaction_distance = 1.5
+    init_pos_x = np.arange(-24, 24, interaction_distance * .9).astype(np.float32)
+    init_pos_y = np.arange(-12, 24, interaction_distance * .9).astype(np.float32)
+    init_pos = np.dstack(np.meshgrid(init_pos_x, init_pos_y)).reshape(-1, 2)
+    domain = np.array([80, 50])
+    pbf = PBF(domain_size=domain, initial_positions=init_pos, interaction_distance=interaction_distance, n_jobs=n_jobs)
```

Comment:
variable `pbf` as an instance of class `PBF` within method `pbf` is not my favorite :)

Reply:
Renamed `PBF` to `PBFSimulator`, so `pbf = PBFSimulator(...)` is hopefully acceptable. :)

Reply:
yesss